Silicon/CommonSocPkg/Include/Library/IgdOpRegionLib.h

```diff
@@ -286,6 +286,13 @@ VOID
   IN CHILD_STRUCT **ChildStructPtr
   );
 
+#pragma pack(1)
+typedef struct {
+  UINTN                   *IgdOpRegionAddressPtr;

```

Comment:
Consider PCD to see if it is a better option.

Reply:
I also thought about a PCD, but I'd like to avoid PcdSet() in a common library. Currently, only core library sets PCDs.

Reply:
what if PCD is set in at individual platform level? will this be better?

Reply:
If much more common libraries are expected, would it be practical to allow common lib use PCD set? After all, platform codes also do set PCDs. Actually i found that in this IgdObRegion lib is using it in function GetVbtAddress ():
_Status = PcdSet32S (PcdGraphicsVbtAddress, VbtAddress);_

Reply:
Right, PCD is updated in platform level, so BootloaderCorePkg and PlatformPkg currently override PCDs. Maybe IgdOpRegionLib can be moved to CommonBoardPkg instead of CommonSocPkg later.

Reply:
Yeah, but that would include moving CML & APL too. V2 will include CFL & CML.